scripts/aggregate-verdict.py

```
@@ -195,6 +195,121 @@ def aggregate(verdicts: list[dict], override: Override | None = None) -> dict:
+    pr_number: str | None = None,
+    head_sha: str | None = None,
+) -> dict:
+    """Generate a quality report from council verdict data."""
+    total = len(verdicts)
+    if total == 0:
+        return {
+            "meta": {
+                "repo": repo,
+                "pr_number": pr_number,
+                "head_sha": head_sha,
+                "generated_at": Path("/tmp").stat().st_mtime if Path("/tmp").exists() else None,
```

Ruff (0.15.0)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)